app/scripts/modules/tencentcloud/src/securityGroup/configure/CreateModal.tsx

```diff
@@ -0,0 +1,281 @@
+import * as React from 'react';
+import {
+  FirewallLabels,
+  ReactModal,
+  WizardModal,
+  WizardPage,
+  noop,
+  TaskMonitor,
+  ReactInjector,
+  SecurityGroupWriter,
+  AccountService,
+  InfrastructureCaches,
+  NameUtils,
+} from '@spinnaker/core';
+import { Subject } from 'rxjs';
+import { get, uniq, intersection } from 'lodash';
+import { ISecurityGroup } from '@spinnaker/core';
+import { ISecurityGroupProps, ISecurityGroupDetail } from '../define';
+import { LoadBalancerLocation } from './components/LoadBalancerLocation';
+import { Ingress } from './components/Ingress';
+export interface ICreateSecurityGroupState {
```

Comment:
It looks like a lot of this state isn't even used.  Please remove any state that is not necessary.  I think most of the state management is handled by the `Location` component.  (See the code in https://github.com/TencentCloud/deck/pull/3 )